lib/Core/BuildEngine.cpp

```diff
@@ -1132,9 +1134,9 @@ class BuildEngineImpl {
       std::string error;
       bool result = db->setCurrentIteration(currentTimestamp, &error);
       if (!result) {
-        // FIXME: Error handling.
-        std::cerr << error << std::endl;
-        exit(1);
+        delegate.error(error);
+        static ValueType emptyValue{};
+        return emptyValue;
```

Comment:
No need to use `static` here, this can just be `return ValueType{}` -- the compiler will make this efficient.